src/fragmentarium/ui/edition/Edition.test.tsx

```diff
@@ -5,28 +5,46 @@ import { Promise } from 'bluebird'
 
 import { submitFormByTestId } from 'test-support/utils'
 import Edition from './Edition'
-import { fragmentFactory } from 'test-support/fragment-fixtures'
+import {
+  fragmentFactory,
+  manuscriptAttestationFactory,
+} from 'test-support/fragment-fixtures'
 import { Fragment } from 'fragmentarium/domain/fragment'
+import { FragmentService } from 'fragmentarium/application/FragmentService'
+import { waitForSpinnerToBeRemoved } from 'test-support/waitForSpinnerToBeRemoved'
+import { ManuscriptAttestation } from 'corpus/domain/manuscriptAttestation'
 
 let fragment: Fragment
+let manuscriptAttestation: ManuscriptAttestation
+let fragmentService
 let fragmentSearchService
 let updateTransliteration
 let container: HTMLElement
 
-beforeEach(() => {
-  updateTransliteration = jest.fn()
-  updateTransliteration.mockReturnValue(Promise.resolve())
+jest.mock('fragmentarium/application/FragmentService')
+
+beforeEach(async () => {
+  updateTransliteration = jest.fn().mockReturnValue(Promise.resolve())
+  fragmentService = new (FragmentService as jest.Mock<
+    jest.Mocked<FragmentService>
+  >)()
   fragmentSearchService = {}
```

Comment:
need to be properly mocked. This should give a typescript error because in Edition Props the type should be `FragmentSearchService`